daprdocs/content/en/reference/components-reference/supported-secret-stores/kubernetes-secret-store.md

```diff
@@ -32,7 +32,12 @@ spec:
 ```
 
 ## Spec metadata fields
-For the Kubernetes secret store component, there are no metadata attributes.
+
+| Field              | Required |  Details | Example |
+|--------------------|:--------:|------------|-----|---------|
+| `defaultNamespace` | N | Default namespace to retrieve secrets from | `"default-ns"` |
```

Comment:
Any consequences if this value is not set?

Reply:
No, it is not a required parameter.